munch-match/src/App.css

```diff
@@ -60,4 +60,9 @@ button {
   bottom: 0px;
   left: 0px;
   z-index: 1;
+}
+
+.range-slider {
```

Comment:
Is this class used anywhere?

Reply:
Yeah, that one overrides some styles in the range-slider class that is auto-generated inside the `RangeSlider`.